src/essreflectometry/amor/__init__.py

```diff
@@ -6,7 +6,7 @@
 import scipp as sc
 
 from .. import providers as reflectometry_providers
-from ..supermirror.types import Alpha, CriticalEdge, MValue
+from ..supermirror import Alpha, CriticalEdge, MValue
```

Comment:
I don't think we should import these like this. `amor.Alpha` is meaningless and could be anything.